GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -457,39 +458,38 @@ private void UpdateStatusOfMenuItems()
             // Many options have no meaning for artificial commits or submodules
             // Hide the obviously no action options when single selected, handle them in actions if multi select
 
-            openWithDifftoolToolStripMenuItem.Enabled = _revisionDiffController.ShouldShowDifftoolMenus(selectionInfo);
-            saveAsToolStripMenuItem1.Visible = _revisionDiffController.ShouldShowMenuSaveAs(selectionInfo);
-            copyFilenameToClipboardToolStripMenuItem1.Enabled = _revisionDiffController.ShouldShowMenuCopyFileName(selectionInfo);
+            // open submodule is added in FileStatusList
+            fileHistoryDiffToolstripMenuItem.Font = selectionInfo.IsAnySubmodule && AppSettings.OpenSubmoduleDiffInSeparateWindow
```

Comment:
This check should use selectedItem instead, that is what FileStatusList uses for the submodule item so two menu items could be bold.
Force pushing